services/simcore/docker-compose.yml

```diff
@@ -19,8 +19,8 @@ services:
           memory: 200M # Determined from prometheus cadvisor actual usage on master, which was <100M
           cpus: '0.5' # For now: same as autoscaling service
         reservations:
-          memory: 200M # Determined from prometheus cadvisor actual usage on master, which was <100M
-          cpus: '0.5' # For now: same as autoscaling service
+          memory: 100M # Determined from prometheus cadvisor actual usage on master, which was <100M
```

Comment:
could you explain the rationale? do you just check below 100M for 10 hours? and then 2x for the limit?

Reply:
As discussed in whereby